packages/react-composites/src/composites/ChatComposite/hooks/useHandlers.ts

```diff
@@ -35,14 +49,15 @@ const createCompositeHandlers = memoizeOne(
       content: string,
       /* @conditional-compile-remove(attachment-upload) */
       options?: {
-        metadata?: Record<string, string>;
         /* @conditional-compile-remove(attachment-upload) */
         attachmentMetadata?: AttachmentMetadata[];
       }
     ) => {
       let metadata = undefined;
       /* @conditional-compile-remove(attachment-upload) */
-      metadata = options?.metadata;
+      metadata = {
```

Comment:
Do we need to set metadate if `JSON.stringify(options?.attachmentMetadata)` returns an empty list?

Reply:
yes that's how contoso can remove attachments, we need to set filesharingMetadata to`'[]'` explicitly